### PR TITLE
make screen reader notification more descriptive, useful

### LINKED ISSUE #235133: Lacking context for question about "screen reader"
Type: <b>Feature Request</b>

VIsual Studio asks me "Are you using a screen reader to operate VS Code?" but does not give any info what it exactly means, what would change for yes/no, why do user sees it in given moment etc. 
I, as a user, have totally no idea what it exactly refers to and how could it help me or contrary - disturb my work.
I don't even see any button to turn off such popup, if I wanted to  mute this.
 
Could you provide some "Help Button" or similar to provide the user with info about this functionality?
Best regards

VS Code version: Code 1.95.3 (f1a4fb101478ce6ec82fe9627c43efbf9e98c813, 2024-11-13T14:50:04.152Z)
OS version: Windows_NT x64 10.0.22631
Modes:

### PR DESCRIPTION
**Pull request by meganrogge**
fix #235133

### CODE REVIEW

File: src/vs/workbench/contrib/accessibility/browser/accessibilityStatus.ts [hunk 1]
```diff
@@ -50,7 +52,7 @@ export class AccessibilityStatus extends Disposable implements IWorkbenchContrib
 	private showScreenReaderNotification(): void {
 		this.screenReaderNotification = this.notificationService.prompt(
 			Severity.Info,
-			localize('screenReaderDetectedExplanation.question', "Are you using a screen reader to operate VS Code?"),
+			localize('screenReaderDetectedExplanation.question', "Are you using a screen reader to operate VS Code? This will enable {0}, optimizing the editor for screen reader usage.", 'editor.accessibilitySupport'),
```
[the remark this answers is not in the document]
Comment (meganrogge) [reply src/vs/workbench/contrib/accessibility/browser/accessibilityStatus.ts:55]: The setting name is very important so users know how to undo this action.

[the remark this answers is not in the document]
Comment (meganrogge) [reply src/vs/workbench/contrib/accessibility/browser/accessibilityStatus.ts]: tweaked it

[the remark this answers is not in the document]
Comment (TylerLeonhardt) [reply src/vs/workbench/contrib/accessibility/browser/accessibilityStatus.ts]: They'd undo it via the status bar entry, no? I have really had to use the setting directly.

File: src/vs/workbench/contrib/accessibility/browser/accessibilityStatus.ts [hunk 2]
```diff
@@ -50,7 +52,7 @@ export class AccessibilityStatus extends Disposable implements IWorkbenchContrib
 	private showScreenReaderNotification(): void {
 		this.screenReaderNotification = this.notificationService.prompt(
 			Severity.Info,
-			localize('screenReaderDetectedExplanation.question', "Are you using a screen reader to operate VS Code?"),
+			localize('screenReaderDetectedExplanation.question', "Screen reader usage detected. Do you want to enable {0} to optimize the editor for screen reader usage?", 'editor.accessibilitySupport'),
```
Comment (TylerLeonhardt) [inline src/vs/workbench/contrib/accessibility/browser/accessibilityStatus.ts:55]:
```suggestion
			localize('screenReaderDetectedExplanation.question', "Screen reader usage detected. Do you want to enable '{0}' to optimize the editor for screen reader usage?", 'editor.accessibilitySupport'),
```

File: src/vs/workbench/contrib/accessibility/browser/accessibilityStatus.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (meganrogge) [reply src/vs/workbench/contrib/accessibility/browser/accessibilityStatus.ts]: We will get feedback from
Users and adjust accordingly

**Review by justschen — APPROVED** [review]

**Pull request merged**